### DISCUSSION TITLE
Error

### DISCUSSION THREAD

**Discussion by Arnav717** [SpotX-Official/SpotX#248] [Errors] [answered]
![image](https://user-images.githubusercontent.com/72515652/206843069-a5ffa984-7d24-4908-9934-568a402d7459.png)
I am getting this error . What should i do?

**Comment by Arnav717** [comment, marked as answer]
also i cant open FAQ , site is not reached

**Reply by amd64fox** [reply]
![photo_2022-12-07_19-20-21](https://user-images.githubusercontent.com/62529699/206843384-edc4705d-2e51-4a0e-bdbf-1487f05b48d4.jpg)

**Reply by Arnav717** [reply]
thanks
